### DISCUSSION TITLE
Where do I find the metadata?

### DISCUSSION THREAD

**Discussion by mindmaze23** [SupaGruen/SD-CheatSheet#4] [General]
Where do I find the metadata?

**Comment by SupaGruen** [comment]
Do you mean an image with metadata?
If so, download this image and test it with that:
https://supagruen.github.io/StableDiffusion-CheatSheet/img/other/metadata-example.jpeg

**Comment by mindmaze23** [comment]
hi
thanks for the reply. I feel stupid but i openning the index, watching the
photos... but I'm missing the metadata\generation info window

**Reply by SupaGruen** [reply]
When you click on the images they should open to reveal all the info.
If you click and nothing happens, then there is something wrong.
